ai-quick-actions/cli.md

```diff
@@ -0,0 +1,821 @@
+
+<details>
+  <summary>CLI Output</summary>
+
+```console
+{
+    "id": "ocid1.datasciencemodeldeploymentint.oc1.iad.<ocid>",
+    "display_name": "MD for falcon7b Model",
+    "aqua_service_model": false,
+    "state": "DELETED",
+    "description": null,
+    "created_on": "2024-02-22 05:00:29.520000+00:00",
+    "created_by": "ocid1.user.oc1..<ocid>",
+    "endpoint": "https://modeldeployment-int.us-ashburn-1.oci.oc-test.com/ocid1.datasciencemodeldeploymentint.oc1.iad.<ocid>",
+    "console_link": "https://cloud.oracle.com/data-science/model-deployments/ocid1.datasciencemodeldeploymentint.oc1.iad.<ocid>?region=us-ashburn-1",
+    "lifecycle_details": "Model Deployment with all associated resources has been deleted",
+    "shape_info": {
+        "instance_shape": "VM.GPU.A10.1",
+        "instance_count": 1,
+        "ocpus": null,
+        "memory_in_gbs": null
+    },
+    "tags": {
+        "OCI_AQUA": ""
+    },
+    "log_group": {
+        "id": "ocid1.loggroup.oc1.iad.<ocid>",
+        "name": "log_name",
+        "url": "https://cloud.oracle.com/logging/log-groups/ocid1.loggroup.oc1.iad.<ocid>?region=us-ashburn-1"
+    },
+    "log": {
+        "id": "ocid1.log.oc1.iad.<ocid>",
+        "name": "llm-deployments",
+        "url": "https://cloud.oracle.com/logging/search?searchQuery=search \"ocid1.compartment.oc1..<ocid>/ocid1.loggroup.oc1.iad.<ocid>/ocid1.log.oc1.iad.<ocid>\" | source='ocid1.datasciencemodeldeploymentint.oc1.iad.<ocid>' | sort by datetime desc®ions=us-ashburn-1"
+    }
+}  
+```
+</details>
+
+
+
+# Model Evaluation
+
+## Create Model Evaluation
+
+### Description
+
+Creates a new evaluation model using an existing Aqua model or model deployment. Currently, evaluation is only supported via model deployment as the source.
+
+### Usage
+
+```bash
+ads aqua evaluation create [OPTIONS]
+```
+
+### Required Parameters
+
+`--evaluation_source_id [str]`
+
+The evaluation source id. Must be either model or model deployment ocid.
+
+`--evaluation_name [str]`
+
+The name for evaluation.
+
+`--dataset_path [str]`
+
+The dataset path for the evaluation. Could be either a local path from notebook session or an object storage path.
+
+`--report_path [str]`
+
+The report path for the evaluation. Must be an object storage path.
+
+`--model_parameters [str]`
+
+The parameters for the evaluation.
+
+`--shape_name [str]`
+
+The shape name for the evaluation job infrastructure.
+
+`--block_storage_size [int]`
+
+The storage for the evaluation job infrastructure.
+
+`--metrics [list]`
+
+The metrics for the evaluation.
+
+
+### Optional Parameters
+
+`--compartment_id [str]`
+
+The compartment OCID where evaluation is to be created. If not provided, then it defaults to user's compartment.
+
+`--project_id [str]`
+
+The project OCID where evaluation is to be created. If not provided, then it defaults to user's project.
+
+`--evaluation_description [str]`
+
+The description of the evaluation. Defaults to None.
+
+`--memory_in_gbs [float]`
```

Comment:
This is only for the flexible shapes. Despite we only support flexible shapes for now, in future we will return the other shapes as well.